source/includes/_tipser-tools.md

```diff
@@ -1,3 +1,35 @@
 # Tipser Tools
 
-//tools desc
\ No newline at end of file
+Tipser enables commerce on almost any site. An easy way to enable shopping on your site is to install the Tipser script and use the Tipser Tools to insert `Get The Look` widget directly into your content. The installation of the script on your site typically takes less than 5 minutes, and can be done by even non-technical people. 
+
+To get Tipser Tools visit [www.tipser.com/pos/tools](https://www.tipser.com/pos/tools) and follow the instructions there.
+
+[![](tipser-tools.png)](/images/tipser-tools.png)
+
+Or, you create an empty bookmark in the bookmarks bar and insert the following script where the URL normally goes:
+
+```
+javascript:(function(){(typeof TipserBookmarklet!='undefined')?TipserBookmarklet.init():document.getElementsByTagName('head')[0].appendChild(document.createElement('script')).src='//www.tipser.com/widget/bookmarklet.js?'+Math.random();}());
+```
+
+Once you have the Tipser Tools on your bookmark bar, simply click it to open on top of any other site. 
```

Comment:
It could be also chrome extension. Maybe that line could be removed ?

Reply:
I think this sentence makes sense when describing bookmarklet.